File: backend/Restaurant.API/Restaurant.Models/Grade.cs
using System;
using System.Collections.Generic;

namespace Restaurant.Models;

public partial class Grade
{
    public int grade_id { get; set; }

    public int rest_gradeid { get; set; }

    public string grade { get; set; } = null!;

    public DateTime grade_date { get; set; }

    public virtual Restaurants RestGrade { get; set; } = null!;
}

File: frontend/Restaurant.UI/src/app/models/grade.ts
export class Restaurant{
    grade_id: number;
    rest_gradeid: number;
    grade: string;
    grade_date: Date;

    constructor (grade_id: number,rest_gradeid: number, grade: string, grade_date: Date){
        this.grade_id = grade_id;
        this.rest_gradeid = rest_gradeid;
        this.grade = grade;
        this.grade_date = grade_date;
    }
}